locations/spiders/dreams_donuts.py

```
@@ -0,0 +1,43 @@
+        "brand": "Dreams Donuts",
+        "brand_wikidata": "Q141142873",
+    }
+    start_urls = ["https://boutiques.dreamsdonuts.com/_next/data/0fZaY95cqqjf19_M8VcLX/index.json"]
+    locations_key = ["pageProps","data","stores"]
+
+    def post_process_item(self, item, response, location):
+        item["name"] = "Dreams Donuts"
+
+        oh = OpeningHours()
+
+        if(location.get("openingHours") != None ):
```

Flake8 (7.3.0)

[error] comparison to None should be 'if cond is not None:'

(E711)